EasyPost.Tests/CreditCardTest.cs

```diff
@@ -36,10 +36,10 @@ public async Task TestFund()
         {
             _vcr.SetUpTest("fund");
 
-            CreditCardFund funding = await CreditCard.Fund("20", CreditCard.Priority.Primary);
+            bool funding = await CreditCard.Fund("20000", CreditCard.Priority.Primary);
 
             Assert.IsNotNull(funding);
```

Comment:
Delete this line.

Rename `funding` to `success` in this unit test to be more explicit about what it is.